bin/kuzzle-createFirstAdmin.js

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env node
+var
+  readlineSync = require('readline-sync'),
+  clc = require('cli-color'),
+  request = require('request-promise'),
+  name,
+  password,
+  step = 0,
+  // steps definitions
+  steps = [
+  	{
+      // get username
+  		prompt: '\n[❓] Which username for the first admin ?\nTip: avoid to name it admin, or root...\n: ',
+  		check: function(userEntry) {
+  			var _name = userEntry.trim(),
+  			badNames = ['admin','root'];
+  			if (badNames.indexOf(_name) === -1) {
+          if (_name.length < 4) {
+            console.log(clc.red('[✖] Username should have at least 4 chars'));
+            return false;
+          }
+
+          name = _name
+  				console.log(clc.green('[✔] Given username: ' + name));
+  				if (_name !== userEntry) {
+  					console.log(clc.cyan('[ℹ] The username have been trimmed.'));
+  				}
+  				name = _name;
+  				return true;
+  			} else {
+  				console.log(clc.red('[✖] It is a bad idea to name an admin "' + _name + '"...'));
+          if (readlineSync.keyInYNStrict('[❓] Are you sure you want to name it "' + _name + '" ?')) {
+            name = _name;
+            return true;
+          }
+  				return false;
+  			}
+  		}
+  	},
+    {
+      // get password
+      password: true,
+      prompt: '\n[?] Which password do you want to set for this account ?\nTips: try to mix lowercased and uppcased letters, digits and special chars. It is always a bad idea to put space char in a password.\n: ',
+      check: function(userEntry) {
+        var _password = userEntry, userEntryBis;
+        if (userEntry.replace('/ /g', '') !== _password || _password.trim() !== userEntry || userEntry.length < 8) {
+          console.log(clc.red('[✖] It looks like your password could be too weak or too difficult to use or type...'));
+          if (userEntry.length > 0) {
+            if (!readlineSync.keyInYNStrict('[❓] Are you sure you want to use this password anyway ?')) {
+              return false;
+            }
+          } else {
+            console.log(clc.red('[✖] Password cannot be empty'));
+            return false;
+          }
+        }
+        userEntryBis = readlineSync.question('\n[?] Could you, please, type it again ?\n: ', {hideEchoBack: true});
+        if (userEntryBis === userEntry) {
+          password = userEntryBis;
+          console.log(clc.green('[✔] Given password set'));
+          return true;
+        } else {
+          console.log(clc.red('[✖] The passwords you typed does not match...'));
+          return false;
+        }
+      }
+    }
+  ];
+
+process.stdin.setEncoding('utf8');
+
+if (process.env.NEW_RELIC_APP_NAME) {
+  require('newrelic');
+}
+
+if (process.env.FEATURE_COVERAGE == 1) {
```

Comment:
Aren't these lines of code used to enable coverage information to be gathered on functional tests?
Are they useful here?